packages/react-search-ui-views/src/MultiCheckboxFacet.js

```diff
@@ -82,14 +87,28 @@ function MultiCheckboxFacet({
       </div>
 
       {showMore && (
-        <button
-          type="button"
-          className="sui-multi-checkbox-facet__view-more"
-          onClick={onMoreClick}
-          aria-label="Show more options"
-        >
-          + More
-        </button>
+        <ScreenReaderStatus
+          render={announceToScreenReader => (
+            <button
+              type="button"
+              className="sui-multi-checkbox-facet__view-more"
+              aria-label="Show more options"
+              onClick={() => {
+                onMoreClick();
+
+                const newLimit = options.length + 10;
```

Comment:
This is a bit of a magic constant (from the parent [Facet.js container](https://github.com/elastic/search-ui/blob/9086bcf5ed438dbbf4d486080b22c704442cf587/packages/react-search-ui/src/containers/Facet.js#L50)) - feel free to shout if you think it should be converted to a prop or something a bit less static